pkg/controllers/controllers.go

```diff
@@ -1 +1,41 @@
 package controllers
+
+import (
+	"net"
+
+	"github.com/coreos/go-iptables/iptables"
+	v1core "k8s.io/api/core/v1"
+)
+
+type IPFamilyHandler struct {
+	Family             v1core.IPFamily
+	IptablesCmdHandler *iptables.IPTables
+	NodeIP             net.IP
+	PodCidr            string
+	BridgeIfName       string
+}
```

Comment:
You should add here `filterTableRules bytes.Buffer` and remove it from `NetworkPolicyController struct` because it is a buffer that contains the iptable rules. You need one for ipv4 and one for ipv6, otherwise the contents get overwritten